src/gui/screen_home.cpp

```diff
@@ -250,13 +251,14 @@ static bool find_latest_gcode(char *fpath, int fpath_len, char *fname, int fname
     return result == FR_OK && fname[0] != 0 ? true : false;
 }
 
-void screen_home_disable_print_button(screen_t *screen) {
-    pw->w_buttons[0].win.f_disabled = 1;
-    pw->w_buttons[0].win.f_enabled = 0; // cant't be focused
-    window_set_text(pw->w_labels[0].win.id, labels[6]);
+void screen_home_disable_print_button(screen_t *screen, int disable) {
+    pw->w_buttons[0].win.f_disabled = disable;
```

Comment:
these f_enabled and f_disabled are very suspicious (at least naming), subject to future refactoring

Reply:
Agreed